exercises/practice/game-of-life/game-of-life.wat

```diff
@@ -0,0 +1,15 @@
+(module
+  (memory (export "mem") 1)
+
+  ;;
+  ;; Create the next frame of Conway's Game of Life
+  ;;
+  ;; @param {i32} offset - The offset of the current frame in linear memory
+  ;; @param {i32} length - The length of the current frame in linear memory
+  ;;
+  ;; @returns {(i32,i32)} - The offset and length of the next frame in linear memory
+  ;;
+  (func (export "next") (param $offset i32) (param $cols i32) (param $rows i32) (result i32)
+    (return (local.get $offset))
```

Comment:
What is being passed in and passed out? The comment and the function signature are not consistent with each other.

Reply:
What is being passed in is now consistent, between the comment and the function signature.

The return is still inconsistent - one i32 or two?  For consistency with other exercises, we should return two: offset and length. (Yes, the length can be deduced, that does not matter. minesweeper and reverse-string and rotational-cipher all return offset and length, even though output length is the same as input length).

I would have expected us to pass in: offset, length, #rows, #columns.   i.e.  number of rows before number of columns.

Reply:
Since the rows and columns are the same as the input, adding the length to the output is totally superfluous. We don't receive a length as input, so the input is already inconsistent - but for a reason.